charts/conf-secrets/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: conf-secrets
 description: A Helm chart for deploying secrets required by MOSIP config-server
 type: application
-version: 0.0.1-develop
+version: 1.3.0
```

Comment:
1.3.0-develop

Also track the changes from last release 1.2.0.2 and mention in README in mosip-infra if any as part of upgrade and get the same added to upgrade document.